### PR TITLE
chore: [IAI-192] Upgrade `react-native` to `0.69.5`

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR aims to upgrade `react-native` to the latest stable version.

## List of changes proposed in this pull request
- Upgraded all native files
- Upgraded libraries if needed
- Patched unsupported libraries

## How to test
Use the app!

### CODE REVIEW

**Review by emiliopavia — APPROVED** [review]
I haven't noticed any regressions on my tests. The only thing to fix is the Podfile that is out of sync after running `pod install`.

File: android/gradle.properties [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
+# Default value: -Xmx512m -XX:MaxMetaspaceSize=256m
+org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m
 
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-android.enableR8=false
+#android.enableR8=false
```
Comment (francescopersico) [inline android/gradle.properties:20]: We can safely remove this

File: android/gradle.properties [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
 org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m
```
Comment (francescopersico) [inline android/gradle.properties:21]: Remove this, we are already setting the jvmargs we need on line 13.

File: android/gradle.properties [hunk 3]
```diff
@@ -30,3 +30,14 @@ android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
 FLIPPER_VERSION=0.154.0
+
+# Use this property to specify which architecture you want to build.
+# You can also override it from the CLI using
+# ./gradlew <task> -PreactNativeArchitectures=x86_64
+reactNativeArchitectures=armeabi-v7a,arm64-v8a,x86,x86_64
+# Use this property to enable support to the new architecture.
```
Comment (francescopersico) [inline android/gradle.properties:38]:
```suggestion

# Use this property to enable support to the new architecture.
```

File: android/gradle.properties [hunk 4]
```diff
@@ -30,3 +30,14 @@ android.enableJetifier=true
...
+# This will allow you to use TurboModules and the Fabric render in
+# your application. You should enable this flag either if you want
+# to write custom TurboModules/Fabric components OR use libraries that
+# are providing them.
+newArchEnabled=false
```
Comment (francescopersico) [inline android/gradle.properties:43]:
```suggestion
newArchEnabled=false

```

**Review by francescopersico — CHANGES_REQUESTED** [review]

**Review by francescopersico — APPROVED** [review]

**Pull request merged**